designer/server/src/models/account/auth.js

```diff
@@ -1,5 +1,17 @@
+import config from '~/src/config.js'
+
 export function signedOutViewModel() {
   return {
     pageTitle: 'You have signed out'
   }
 }
+
+/**
+ *  @param {boolean} userFailedAuthorisation - indicates the user has failed authorisation, but authentication was successful
+ */
+export function signInViewModel(userFailedAuthorisation) {
+  return {
+    pageTitle: `Sign in to ${config.serviceName}`,
+    userFailedAuthorisation
```

Comment:
Not a blocker, but we could ahead and build the `errorList` array here

Making the Nunjucks view nice and generic for shared usage

Reply:
Done